about.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  
+  <!-- Bootstrap CSS -->
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="stylesheet" href="assets/css/styles.css">
+  <!-- Cocogoose font family -->
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <!-- Lato font family -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap" rel="stylesheet">
+  <title>Art Selection welcome page</title>
+</head>
+<body>
+  <header>
+    <!--ADD THE FIRST LEVEL -->
+    <div class="container-fluid display-desktop social" id="social">
+      
+    </div>
+    <!--ADD THE NAVBAR -->
+    <nav class="navbar navbar-expand-md navbar-light bg-light sticky-top" id="my-navbar">
+      
+    </nav>
+  </header>
```

Comment:
- [ ] Kindly apply the suggestion from home page on this page too.
- [ ] There's no menu option to navigate to the home page from about page on the mobile version like your did on the home page when the hambuger icon is clicked. Kindly look into that and fix.

### Screen Recording
[your-about-to-home-missing.webm](https://user-images.githubusercontent.com/26597432/235002856-c861a64f-9e24-4c1c-965a-ad0290f0d7f3.webm)

Reply:
Done @Qoosim